incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -470,81 +489,89 @@ would require them to set the `allowCascadingDeletion` property of the child
 namespace.
 
 <a name="admin-types"/>
+<a name="admin-resources"/>
 
 ### Modify the object types propagated by HNC
```

Comment:
Modify the object **resources** propagated by HNC?

Reply:
Done